src/Microsoft.PowerApps.TestEngine/Config/TestSettingExtensionSource.cs

```diff
@@ -8,14 +8,21 @@ public class TestSettingExtensionSource
     {
         public TestSettingExtensionSource()
         {
-            EnableFileSystem = true;
+            EnableFileSystem = false;
             InstallSource.Add(Path.GetDirectoryName(this.GetType().Assembly.Location));
         }
 
+#if RELEASE
+        //restricting for current milestone
```

Comment:
I am not super fan of this type of comments, because they are hard to track and not meaningful unless somebody have clear understanding of the milestones, and what is each milestone about. Is it that they are needed for identify code that we need to revisit in the future?  Could it better to have // TODO: ... ? but neither I am fan of todo comments, hard to track with work items.

Reply:
can generate workitem to track this and remove the comment

Reply:
created issue #503